.devcontainer/build_cxx.sh

```
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -ev
+
+NPROC=$(nproc --all)
+SCRIPT_PATH=$(dirname $(realpath -s $0))
```

Shellcheck

[warning] Quote this to prevent word splitting.

(SC2046)